pkg/monitoring/metrics/rbd_metrics.go

```diff
@@ -0,0 +1,68 @@
+package metrics
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/machadovilaca/operator-observability/pkg/operatormetrics"
+	k8sv1 "k8s.io/api/core/v1"
+	kubevirtv1 "kubevirt.io/api/core/v1"
+)
+
+var (
+	rbdMetrics = []operatormetrics.Metric{
+		vmRbdVolume,
+	}
+
+	vmRbdVolume = operatormetrics.NewGaugeVec(
+		operatormetrics.MetricOpts{
+			Name: metricPrefix + "vm_rbd_volume",
+			Help: "VM with RBD mounted volume",
+			ExtraFields: map[string]string{
+				"StabilityLevel": "ALPHA",
+			},
+		},
+		[]string{"name", "namespace", "pv_name", "volume_mode", "rxbounce_enabled"},
+	)
+)
+
+func SetVmWithVolume(vm *kubevirtv1.VirtualMachine, pvc *k8sv1.PersistentVolumeClaim, pv *k8sv1.PersistentVolume) {
+	if pv.Spec.PersistentVolumeSource.CSI == nil {
```

Comment:
Could be combined with following if-block.

Reply:
okay